index.bs

```diff
@@ -87,20 +87,20 @@ interface VRDisplay : EventTarget {
   readonly attribute DOMString displayName;
 
   /**
+   * Populates the passed VRFrameData with the information required to render
+   * the current frame.
+   */
+  void getCurrentFrameData(VRFrameData frameData);
```

Comment:
With the OpenVR SDK (and I *think* Oculus, but I'm not 100% certain) there
are times where you can't poll a valid pose but do know that the hardware
is connected. This happens mostly during initialization of the associated
runtime or when the headset become completely occluded. It may also be
necessary to artificially prevent the frame data from being populated for
privacy or security concerns, which I would see as another valid reason for
returning false.

On Sat, Sep 10, 2016, 1:02 PM Rafael Cintron <notifications@github.com>
wrote:

> In index.bs <https://github.com/w3c/webvr/pull/80#discussion_r78283194>:
>
> > @@ -87,20 +87,20 @@ interface VRDisplay : EventTarget {
> >    readonly attribute DOMString displayName;
> >
> >    /**
> > +   * Populates the passed VRFrameData with the information required to render
> > +   * the current frame.
> > +   */
> > +  void getCurrentFrameData(VRFrameData frameData);
>
> @Toji <https://github.com/Toji>, under what circumstances would
> getFrameData return false?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/w3c/webvr/pull/80/files/e6c83cb18cccf995d70b83ae2e0add6ef3b7a59d#r78283194>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAxJmVIN4dp8PnI2aJDd-j1KTuQjoMrqks5qowzSgaJpZM4J2KLS>
> .
>